docs/pull_request_template.md

```diff
@@ -6,4 +6,13 @@
 <!-- What are the reasons behind this change being made? -->
 
 ## Visual Changes
-<!-- If the change results in visual changes, show a before and after -->
+<!-- If change results in visual changes, include detailed screenshots that show the various states. -->
+
+<!-- Please ensure that the changes are reviewed by a Designer if required. -->
+<!-- To help Designers, please include a link to specific elements to review, -->
+<!-- for example to https://components.publishing.service.gov.uk/public -->
```

Comment:
The preview URL pattern has been updated so it's now predictable - could this be included in the example link?

```suggestion
<!-- for example to https://components-gem-pr-[PULL REQUEST NUMBER].herokuapp.com/public -->
```